Sources/Gui/Main.cpp

```diff
@@ -57,13 +57,23 @@
 #endif
 
 
+#ifdef _MSC_VER>=1900	// Visual Studio 2015 or higher
+extern "C" { FILE __iob_func[3] = { *stdin,*stdout,*stderr }; }
```

Comment:
You are forgetting to put `libpng16-16.dll` (which can be found in [the OpenSpades binary release](https://github.com/yvt/openspades/releases/tag/v0.0.12)) into the same directory as the executable file.